### PR TITLE
Giving user boundary to help with indentation issue. 

### PR DESCRIPTION
**Pull request by prastut**

### CODE REVIEW

File: src/templates/chapter.js [hunk 1]
```diff
@@ -366,6 +366,8 @@ const ChapterTemplate = ({ data: { mdx: chapter } }) => {
               fontFamily: "'Inconsolata', monospace",
               wordWrap: true,
               wordBasedSuggestions: false,
+              autoIndent: true,
```
Comment (bhaskarSingh) [inline src/templates/chapter.js:369]: Screenshot 2020-05-22 at 8 01 48 PM

autoIndent doesn't support 'true' option

File: src/templates/chapter.js [hunk 2]
```diff
@@ -366,6 +366,8 @@ const ChapterTemplate = ({ data: { mdx: chapter } }) => {
...
+              formatOnType: true,
```
Comment (bhaskarSingh) [inline src/templates/chapter.js:370]: What does `formatOnType` does?

File: src/templates/chapter.js [hunk 3]
```diff
@@ -400,6 +402,7 @@ const ChapterTemplate = ({ data: { mdx: chapter } }) => {
                     fontFamily: "'Inconsolata', monospace",
                     renderSideBySide: false,
                     wordWrap: true,
+                    renderIndicators: false,
```
Comment (bhaskarSingh) [inline src/templates/chapter.js:405]: By default `renderIndicators` option is set to `true`, no need for explicitly defining it.

**Comment by prastut** [comment]
@bhaskarSingh I have reverted all editor changes.

**Comment by bhaskarSingh** [comment]
> I have reverted all editor changes.

Got it, I have added feedback.

**Comment by bhaskarSingh** [comment]
@prastut  I am pushing the changes myself, closing the issue in reference to the slack message for sending feedback to tezos foundation.

**Pull request merged**